WordPress/src/main/java/org/wordpress/android/ui/media/MediaGridAdapter.java

```diff
@@ -289,7 +289,6 @@ private void loadNetworkImage(Cursor cursor, NetworkImageView imageView) {
             String filepath = uri.getLastPathSegment();
 
             int placeholderResId = WordPressMediaUtils.getPlaceholder(filepath);
-            imageView.setImageResource(0);
             imageView.setErrorImageResId(placeholderResId);
```

Comment:
I'm not sure why `setErrorImageResId` is called here, I think we should move it outside the ` if (thumbnailURL != null)`